CRM/PivotCache/GroupActivity.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Manages 'pivotreport' cache group.
+ */
+class CRM_PivotCache_GroupActivity extends CRM_PivotCache_AbstractGroup {
+
+  public function __construct($name = NULL) {
+    $this->name = 'pivotreport.Activity';
```

Comment:
Given that the AbstractGroup class already creates the name by prefixing it with `pivotreport`, what about doing this instead?

```php
parent::__construct('Activity');
```